test/Prototypes/Unicode.swift

```diff
@@ -63,9 +99,12 @@ final class _StringStorage<Element: UnsignedInteger>
       body(UnsafeBufferPointer(start: UnsafePointer($0.baseAddress), count: count))
     }
   }
-/*}
+/*
+}
+
+extension _StringStorage : _NSStringCore {
+*/
 
-extension _StringStorage : _NSStringCore {*/
```

Comment:
I don't recall the issue; let me try to repro once I get a clean build. My changes were adjusting the comments for my own visual sanity, but were probably needless churn. Since this is just a prototypes file, it's kind of a dumping ground and the last person to touch it is the style dictator.